README.md

```diff
@@ -39,6 +39,9 @@ Notes regarding Python 3.9 and Conda:
    1. Update to latest `pip`:
       * Linux: `.venv/bin/python -m pip install --upgrade pip`
       * Windows: `.venv\Scripts\python -m pip install --upgrade pip`
+    1. Update to latest `wheel`:
+      * Linux: `.venv/bin/python -m pip install --upgrade wheel`
+      * Windows: `.venv\Scripts\python -m pip install --upgrade wheel`
```

Comment:
are you sure we need `wheel` to run the examples? is the "speed up" related to wheel being able to cache some locally built packages?

Reply:
It's not needed but the install of the requirements files seems faster with it. It's not mandatory, so we can remove it if you prefer.